data/gallery.js

```diff
@@ -230,10 +238,10 @@ function handleHoverOut(li) {
 }
 
 /**
- * Run the pageun start here
+ * Run the page start here
  */
 setupPageElements();
-album.images.forEach(logImages);
+album.images.forEach(loadAlbumlbumIages);
 document.body.appendChild(gallery);
```

Comment:
tiny (mac based keyboard) issue with the name here:
loadAlbumlbumIages probably was meant to be loadAlbumImages